mongonaut/templates/mongonaut/base.html

```diff
@@ -10,62 +10,68 @@
 
 
 <body>
-<ul class="breadcrumb">
-    {% block breadcrumbs %}
-    {% endblock %}
-    <li style="float:right;"><a href="{% url 'admin:logout' %}" class="active">Logout</a></li>
-</ul>
-<div class="container-fluid">
 
-    {% if messages %}
-    <div class="alert-messages">
-        {% for message in messages %}
-        <div class="alert {% if message.tags %} alert-{{ message.tags }} {% endif %}">
-            <a class="close" data-dismiss="alert">&times;</a>
-            <p>{{ message }}</p>
-        </div>
-        {% endfor %}
-    </div>
-    {% endif %}
+    <div class="container">
 
-{% block content %}{% endblock %}
-    <div class="row">
-        <div class="span12">
-            <footer>
-                <p>
-                    &copy; 2012 <a href="//pydanny.com/category/blog.html">Daniel Greenfeld</a>
-                    - <a href="//github.com/jazzband/django-mongonaut">Fork on Github!</a>
-                    - <a href="//django-mongonaut.rtfd.org">Documentation</a>
-                </p>
-            </footer>
-        </div>
-    </div>
-</div>
+        <!-- <div class='row'>  -->
+            <ol class='breadcrumb'>
+            {% block breadcrumbs %}
+            {% endblock %}
+            </ol>
+        <!-- </div> -->
 
-{% block extrajs %}
-{% endblock %}
 
-<script type="text/javascript">
-//<![CDATA[
-{% block inlinejs %}
-{% endblock %}
-//]]>
-</script>
+        <div class="container-fluid">
 
-<script type="text/javascript">
-    $("#action_delete").click(function(){
-        $('.btn-group').hide();
-        $('#delete_warning').show();
-    });
-    $(".alert_close").click(function(){
-        $('.btn-group').show();
-        $('#delete_warning').hide();
-    });
-    $("#action_delete_ok").click(function(){
-        var form_ = $('#delete_form');
-        form_.submit();
-    });
-</script>
+            {% if messages %}
+            <div class="alert-messages">
+                {% for message in messages %}
+                <div class="alert {% if message.tags %} alert-{{ message.tags }} {% endif %}">
+                    <a class="close" data-dismiss="alert">&times;</a>
+                    <p>{{ message }}</p>
+                </div>
+                {% endfor %}
+            </div>
+            {% endif %}
 
-</body>
+        {% block content %}{% endblock %}
+<!--             <div class="row">
```

Comment:
As free and open source software this should stay in the displayed app. If someone doesn't want it Django allows overriding of templates.  

At the very least it should be updated to point to the new location of the code.